WordPress/src/main/java/org/wordpress/android/ui/people/PersonActivity.java

```diff
@@ -17,14 +17,17 @@
 
 public class PersonActivity extends AppCompatActivity {
     public static final String EXTRA_PERSON_ID = "EXTRA_PERSON_ID";
+    public static final String EXTRA_LOCAL_BLOG_ID = "EXTRA_LOCAL_BLOG_ID";
 
-    private int mPersonId;
+    private long mPersonId;
+    private int mLocalBlogId;
```

Comment:
Same comment here as in `PeopleManagementActivity.java` about perhaps demoting these variables to local ones instead of global. Cheers!